setup/bootstrap_docker.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+set -eu
+
+FILES_BASE_URL=/opt/redash/current/setup/files/
+
+# Base packages
+apt-get dist-upgrade
+apt-get update
+apt-get install -y python-pip python-dev curl build-essential pwgen libffi-dev sudo git-core
+pip install -U setuptools
+
+# redash user
+useradd --system --comment " " --create-home redash
+
+# PostgreSQL
+apt-get update
+apt-get -y install libpq-dev postgresql
+
+# Redis client
+mkdir -p /tmp/redash && cd $_
+curl -L https://github.com/antirez/redis/archive/3.0.4.tar.gz -o redis.tar.gz
+mkdir redis
+tar -zxvf redis.tar.gz -C redis --strip-components 1
+cd redis
+make redis-cli
+cp src/redis-cli /usr/local/bin
+rm -rf /tmp/redis
```

Comment:
Are you sure we need redis installed, to install the Redis Python package?

Because otherwise we don't need the Redis client on this image.